pkg/telemetry/gather.go

```diff
@@ -233,3 +240,38 @@ func fetchKubernetesResources(ctx context.Context, cfg *config.Config) (map[stri
 	}
 	return metrics, nil
 }
+func getOSVersion() (string, error) {
```

Comment:
Is this code more "appropriate" for the util package?

Reply:
Makes sense, will move it.